Examples/PcapSplitter/main.cpp

```diff
@@ -184,6 +184,109 @@ std::string getFileNameWithoutExtension(const std::string& path)
 	return("");
 }
 
+std::string getIP(pcpp::Packet& packet, bool client)
+{
+	std::string packetStr = packet.printToString();
+	int srcPos = packetStr.find("IPv4") + 17;
+	int separator = packetStr.find(",", srcPos + 1);
+	std::string srcIP = packetStr.substr(srcPos, separator - srcPos);

```

Comment:
Why extracting source and dest IP like this?
It's better to do something like this:

    pcpp::IPv4Layer* ipLayer = packet.getLayerOfType<IPv4Layer>();
    if (ipLayer == NULL)
        return "";
    std::string srcIP = ipLayer->getSrcIpAddress().toString();
    std::string dstIP = ipLayer->getDstIpAddress().toString();